Add access checks for several checks

zfLQ2qx2 opened the pull request:
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

@toniblyx The other issue raised in issue 617 is that the reporter was using a policy that didn't allow cloudtrail describe-trails and that led to unexpected results. That seems to be a common theme in the issues section and I think the only real fix is to be defensive and look for access denied errors whenever an aws cli command is run. This adds a number of AccessDenied checks in the check_2x checks and a couple extra_7xx checks that happen to call describe-trails.

toniblyx reviewed (changes requested):
@zfLQ2qx2 thanks for adding this better way of handling AccessDenied. I added CHECK_SERVICENAME variable to each check, could you add it to the checks before I merge it? See any check in master for reference.

zfLQ2qx2 commented:
> @zfLQ2qx2 thanks for adding this better way of handling AccessDenied. I added CHECK_SERVICENAME variable to each check, could you add it to the checks before I merge it? See any check in master for reference.

@toniblyx More to come! I pulled your latest changes and rebased, I spot checked three of them and the CHECK_SERVICENAME_ tags are now there, so I think this is ready for merging.
